### PR TITLE
Add Take All Button

### PR DESCRIPTION
**Pull request by anubisthejackle**
As mentioned in Issue #200 

This PR adds a Take All button, which will take as much as it can, and if that takes everything, it will "leave" if it is not everything, it launches the Drop menu, and stops attempting to take items.

Updated translation files to match changes.

### CODE REVIEW

File: lang/es/strings.po [hunk 1]
```diff
@@ -1279,131 +1279,139 @@ msgstr "recuperar más salud al comer"
...
 msgstr "aprendido a obtener lo máximo de la comida"
 
-#: ../../script/engine.js:137 ../../script/engine.js:473
+#: ../../script/engine.js:138 ../../script/engine.js:485
 msgid "lights off."
 msgstr "luces fuera."
 
-#: ../../script/engine.js:143
+#: ../../script/engine.js:144 ../../script/engine.js:150
+msgid "hyper."
```
Comment (Continuities) [inline lang/es/strings.po]: Weren't these translations included in other PRs?

File: script/events.js [hunk 2]
```diff
@@ -503,7 +503,41 @@ var Events = {
...
+				text: _('Take All'),
+				click: Events.takeAllLoot
+			}).addClass('take-all-button').appendTo(lootButtons);
+		}
+	},
+
+	takeAllLoot: function(){
+
+		var stoppedEarly = false;
+		$('#lootButtons')
```
Comment (Continuities) [inline script/events.js]: If I'm reading this right, this button simulates a single click on each loot button. That means it'll still require multiple clicks to truly "take all," no?

**Comment by Continuities** [comment]
I'm seeing a bunch of translation string changes that I thought were in previous PRs: text for 'hyper mode' and stuff.

**Comment by anubisthejackle** [comment]
Since I added the Take All text, I re-ran poedit and it updated the line numbers it seems. As for the click, looking over the code that's exactly correct. I'm noticing the glitch was that it didn't actually take all it just didn't end early, so closed early. I'll fix that up.

**Comment by anubisthejackle** [comment]
I had plotted but not actually written that last commit it seems. That's what I get for distracted coding.

**Comment by skj3gg** [comment]
Sorry I hadn't tested this out yet. Was that the reason this was closed?

**Comment by anubisthejackle** [comment]
No, merge conflicts. I'll reopen after I work them out.

**Comment by skj3gg** [comment]
Cool. I'll pull it down and run through the game over the next couple days, promise.

**Comment by anubisthejackle** [comment]
Sounds good. Still saying conflicts after merging master in, so I'm going to assume it's a line ending thing? Had no conflicts with merging in master.

**Comment by anubisthejackle** [comment]
Turns out I'd forgotten to change my upstream after the name change. Woops.

**Comment by skj3gg** [comment]
8c368d363ab6568dc3d68faa76cdc7565a54221b finally got to it!

I amended some small clean-up css and changed text to lowercase.

Thanks!

**Comment by youknowone** [comment]
I am watching problems of this button.

1. The behavior is not very clear. When I have 7 spaces and 7 cured meat, it didn't pick up meats. Even more, it sometimes pick only 4 or 2 or any way a few less than how much i can get.
2. When I tried 'take all' in dungeon (like city), the character went out of the dungeon, even though i didn't choose any continue/leave button
3. Alignment of this button is very weird

This button is dropping usability for now, because 1 or 2 is not predictable mostly.

**Comment by AndySky21** [comment]
I haven't tested the button yet. I'm using my version of ADR and I have messed with it wildly. So I have to try a Master.
However, I have to suggest a thing. It'd be better if the button only "takes all", so that the player has the idea of what has happened, then s/he can decide to leave.

**Comment by AndySky21** [comment]
I definitely agree with @youknowone. Take all doesn't work properly, and what's more, it often prevents completing dungeons!
Can I try to improve it?

**Comment by AndySky21** [comment]
Ok, I'm unreliable and now you know it.
I said I was going to do nothing, instead I PRed the big one. #399 tries to fix stuff from Take everything and leave, by not actually "leaving" if there are more than one option, and by giving the player the option to take "all (possible) of a kind". There are also some minor fixes to drop menu.

**Pull request closed without merging**